util/additional-measures.json

```diff
@@ -80,5 +80,33 @@
     "measureSets": [],
     "isBonus": false,
     "objective": "electronicPrescribing"
-  }
+  },
+  {
+    "category": "aci",
+    "firstPerformanceYear": 2017,
+    "lastPerformanceYear": null,
+    "metricType": "boolean",
+    "measureId": "ACI_LVSSC_1",
+    "title": "Low-Volume Send a Summary of Care",
```

Comment:
Is there an email where these titles are coming from? I'm finding the titles confusing. What about "Low-Volume Outgoing Transfer of Care" and "Low-Volume Incoming Transfer of Care"